src/utils/container.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * Checks if the list includes any value from the search.
+ * @param list The list to search in.
+ * @param search The values to search.
+ * @returns {boolean} _true_ if the list includes any value from the search.
+ */
+export function includesAny(list: string | string[] | null | undefined, search: string | string[]) {
+    if (!list) {
+        return true;
+    }
+
+    let negative = false;
```

Comment:
`negative` seems a bit ambiguous... maybe `inverseMatch` or something similar would be more clear? :man_shrugging: